### DISCUSSION TITLE
Docmost vs. Outline

### DISCUSSION THREAD

**Discussion by ReessKennedy** [docmost/docmost#467] [Help / Questions]
Anyone used both and have any thoughts on how they differ? Ways Docmost has a different approach? 

I have used Outline but not Docmost and curious.

**Comment by thalion0** [comment]
i've just tested outline. Outline is great and some features ahead. Docmost, correct me if am wrong, is more efficient and easier in case of setup and UI. Missing ability to create more than one workspaces in community version caused be to use docmost. Additional in outline you are not able to create a diagrams.net out of document as docmost do.
